### PR TITLE
kics: init at 1.5.12

### PR DESCRIPTION
**Pull request by patryk4815**
###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [X] aarch64-linux
  - [X] x86_64-darwin
  - [X] aarch64-darwin
- [X] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by legendofmiracles** [comment]
Is your nixpkgs revision old? When I check all-packages.nix the default version of buildGoModule is 1_18

File: pkgs/tools/admin/kics/default.nix
Comment (azahi) [inline pkgs/tools/admin/kics/default.nix:16]: Please provide a comment on why is this disabled.

File: pkgs/tools/admin/kics/default.nix
Comment (azahi) [inline pkgs/tools/admin/kics/default.nix:27]: Please add `-s` and `-w` flags.

File: pkgs/tools/admin/kics/default.nix
Comment (azahi) [inline pkgs/tools/admin/kics/default.nix:33]:
```suggestion
    description = ''
      Find security vulnerabilities, compliance issues, and infrastructure misconfigurations early in the development
      cycle of your infrastructure-as-code with KICS by Checkmarx.
    '';
```

**Review by azahi — CHANGES_REQUESTED** [review]
Please split your commits into two separate ones: adding yourself as a maintainer and initializing the package. See [this](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes).

File: pkgs/tools/admin/kics/default.nix
Comment (patryk4815) [reply pkgs/tools/admin/kics/default.nix]: done :)

File: pkgs/tools/admin/kics/default.nix
Comment (patryk4815) [reply pkgs/tools/admin/kics/default.nix]: removed, it was copied from different pkg.

File: pkgs/tools/admin/kics/default.nix
Comment (patryk4815) [reply pkgs/tools/admin/kics/default.nix]: done

**Comment by patryk4815** [comment]
@azahi now better 🎉 ?

**Review by azahi — APPROVED** [review]

**Review by arcz — APPROVED** [review]

**Review by winterqt — CHANGES_REQUESTED** [review]
Your commit message should read `init at 1.5.12`, without the `v` prefix. LGTM otherwise, will merge after this is fixed.

**Comment by patryk4815** [comment]
@winterqt fixed

**Review by winterqt — APPROVED** [review]

**Pull request merged**
